telegram/_chatadministratorrights.py

```diff
@@ -32,13 +32,12 @@ class ChatAdministratorRights(TelegramObject):
     :attr:`can_post_messages`, :attr:`can_edit_messages`, :attr:`can_pin_messages`,
     :attr:`can_manage_topics` are equal.
 
+    .. seealso:: :meth:`Bot.set_my_default_administrator_rights`
```

Comment:
why did you drop `get_my_…`?

Reply:
because it's in the autogenerated "Returned in" admonition